docs/pages/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -215,4 +215,4 @@ is emitted to the Audit Log.
 - TLS protocol version is restricted to TLS 1.2 and TLS 1.3.
 - All uses of non-compliant algorithms such as NaCl are removed and replaced with compliant algorithms such as AES-GCM.
 - Teleport is compiled with [BoringCrypto](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4407).
-- User, host, and CA certificates (and host keys for recording proxy mode) only use 2048-bit RSA private keys.
+- User, host, and CA certificates (and host keys for recording proxy mode) only use 2048-bit, 3072-bit, or 4096-bit RSA private keys.
```

Comment:
BoringCrypto may not prevent it, but we still exclusively use 2048-bit RSA keys (at least until we get around to implementing [RFD 136](https://github.com/gravitational/teleport/blob/master/rfd/0136-modern-signature-algorithms.md))